OpenRA.Mods.Common/Traits/Infantry/ScaredyCat.cs

```diff
@@ -82,12 +82,14 @@ public void Damaged(Actor self, AttackInfo e)
 				Panic();
 		}
 
-		public void Attacking(Actor self, Target target, Armament a, Barrel barrel)
+		void INotifyAttack.Attacking(Actor self, Target target, Armament a, Barrel barrel)
```

Comment:
The idea is (or was) that a scared actor would have a certain probability of panicking when attacking, I guess (in the original games, civilians that were shot at would run around in panic between attacks even while attacking a target). Not sure if that still works, but I consider that out of scope for this PR either way.